recipes-server/nginx/nginx_1.21.1.bbappend

```diff
@@ -11,7 +11,6 @@ do_install:append () {
 
     install -Dm 0644 ${WORKDIR}/default ${D}${sysconfdir}/nginx/sites-available/default_server
     sed -i 's,/var/,${localstatedir}/,g' ${D}${sysconfdir}/nginx/sites-available/default_server
-    install -d ${D}${sysconfdir}/nginx/sites-enabled
```

Comment:
and squash to nginx?

Reply:
It is too late to squash a part of mid-places commit into the  nginx one, I'm afraid